pages/Home.vue

```diff
@@ -38,8 +38,6 @@
       </template>
     </SfCallToAction>
 
-    <newsletter-popup v-if="loadNewsletterPopup" />
```

Comment:
What about `SfCallToAction` _"Subscribe to Newsletters"_ above? If there is no newsletter support then maybe it also should be removed 🤔

Reply:
In my opinion, we shouldn't' because it fills up the home page and this looks good for potential clients. This feature should not works properly at this time but in my opinion, it should stay.

Reply:
:+1: